source/configure/elasticsearch-configuration-settings.rst

```diff
@@ -502,15 +502,79 @@ Live indexing batch size
 *Available in legacy Enterprise Edition E10/E20*
 
 +---------------------------------------------------------------+-----------------------------------------------------------------------------------+
-| The number of new posts batched together before they're       | - System Config path: N/A                                                         |
-| added to the Elasticsearch index.                             | - ``config.json`` setting: ``".Elasticsearchsettings.LiveIndexingBatchSize: 1",`` |
-|                                                               | - Environment variable: ``MM_ELASTICSEARCHSETTINGS_LIVEINDEXINGBATCHSIZE``        |
-| Numerical input. Default is **1**.                            |                                                                                   |
+| The number of new posts needed before they're added to the    | - System Config path: N/A                                                         |
+| Elasticsearch index.                                          | - ``config.json`` setting: ``".Elasticsearchsettings.LiveIndexingBatchSize: 1",`` |
+| Once added to the Index the post becomes searchable.          | - Environment variable: ``MM_ELASTICSEARCHSETTINGS_LIVEINDEXINGBATCHSIZE``        |
+| On servers with more than one post per second, we suggest     |                                                                                   |
+| this to the average posts per 20 seconds.                     |                                                                                   |
+|                                                               |                                                                                   |
+| Numerical input. Default is **1**. Meaning an index job is    |                                                                                   |
```

Comment:
No job is actually run. Perhaps better to say "every post is indexed synchronously as they are created".

Reply:
@coltoneshaw - I'll apply this update. Thanks, @agnivade!